app/models/manageiq/providers/ibm_cloud/object_storage/object_manager.rb

```diff
@@ -0,0 +1,211 @@
+class ManageIQ::Providers::IbmCloud::ObjectStorage::ObjectManager < ManageIQ::Providers::StorageManager
```

Comment:
I think we should follow the other (power and vpc) and go with Vendor::Service::ManagerType so e.g. `ManageIQ::Providers::IbmCloud::Vpc::CloudManager` which would mean this would be `ManageIQ::Providers::IbmCloud::ObjectStorage::StorageManager`.

We don't have an `ObjectManager` type which makes this confusing to me at least.

The only other pattern is when there are multiple types of storage managers (e.g. OpenStack/Amazon) which does:
`ManageIQ::Providers::Amazon::StorageManager::Ebs/S3`  so it is still obvious it is a Storage Manager.

Reply:
@iv1111 I put together a rename commit here (specs even pass, yea! :smile:):
https://github.com/iv1111/manageiq-providers-ibm_cloud/compare/cloud_object_storage...jaywcarman:cloud_object_storage_rename_manager

Also included is a commit for #202

Reply:
👍 that looks good @jaywcarman

Reply:
@agrare Our COS provider class is called ManageIQ::Providers::IbmCloud::ObjectStorage::Object**Manager**. We had to call it like that, since the UI code parses the classname in order to set the "kind" fild . It strips away the "Manager" part and leaves whatever is in front of it setting the "kind" value. Filtering by the value of "kind" is necessary, so that one would display only ObjectStorage providers supported by MIQ (and not e.g. Block Storage providers).

Reply:
Oh you had to name it `ObjectManager`?  That's not good, block storage managers aren't named `BlockManager`.  I thought you were checking the `supports_object_storage?` feature.

Reply:
@agrare Yes, that's the reason. We were not able to use _supports_object_storage?_, because current API would not let one to get that value for a provider.